### DISCUSSION TITLE
Use UTF-8 encoding in core_metrics

### DISCUSSION THREAD

**Discussion by rolandramp** [sebastian-hofstaetter/teaching#142] [Advanced IR: Exercise Q&A]
I had some problems loading the fira-22.baseline-qrels.tsv, so I did a: "with open(path, 'r', encoding="utf-8")" in the load_qrels function to avoid encoding problems. But maybe this is only a problem, because I am working on a Windows PC.

**Comment by ldosen00** [comment]
Feel free to state which specific problems you had, but specifically stating the encoding parameter is quite normal.
